src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { mergeMap } from 'rxjs/operators';
+import { CmsService, HalDoc } from '../core';
+import { SeriesModel } from '../shared';
+
+@Component({
+  selector: 'publish-dashboard',
+  styleUrls: ['dashboard.component.css'],
+  templateUrl: 'dashboard.component.html'
+})
+
+export class DashboardComponent implements OnInit {
+
+  isLoaded = false;
+  totalCount: number;
+  noSeries: boolean;
+  auth: HalDoc;
+  account: HalDoc;
+  series: SeriesModel[];
+
+  constructor(private cms: CmsService) {}
+
+  ngOnInit() {
+    this.isLoaded = false;
+    this.cms.auth.pipe(
+      mergeMap((auth: HalDoc) => {
+        this.auth = auth;
+        return auth.follow('prx:default-account');
```

Comment:
TODO: defaultAccount for stories without series(pretty sure?) but the series has a `prx:account` and not just the default

Reply:
fixed